soc/xtensa/espressif_esp32/esp32s3/default.ld

```diff
@@ -233,9 +233,20 @@ SECTIONS
   {
     _ext_ram_bss_start = ABSOLUTE(.);
     *(.ext_ram.bss*)
-    . = ALIGN(0x10000);
+    . = ALIGN(4);
```

Comment:
This alignment was previously wrong / just arbitrary?

Reply:
It was arbitrary, for testing purposes. Ended up forgetting to put it on the first commit. Thanks for pointing it out